.circleci/config.yml

```diff
@@ -2,39 +2,45 @@ version: 2
 jobs:
   unit-php72-magento23:
     docker:
-      - image: boltdev/m2-plugin-ci-php72:v1
+      - image: boltdev/m2-installed-plugin-ci-php72:2.3.0
```

Comment:
nit: `boltdev/m2-230-installed-plugin-ci-php72:v1` the reason being the tag is our version not the magento version

for all the images

Reply:
We need to tag the Magento version because we can have multiple magento versions tied to a given php instance.  Would you prefer the tag to be 2.3.0-v1